core/src/main/scala/sbt/librarymanagement/CrossVersionExtra.scala

```diff
@@ -5,19 +5,23 @@ import sbt.librarymanagement.syntax._
 
 final case class ScalaVersion(full: String, binary: String)
 
-private[librarymanagement] abstract class CrossVersionFunctions {
+private[librarymanagement] trait CrossVersionCompatAliases { self: CrossVersionFunctions =>
 
   /** Compatibility with 0.13 */
-  final val Disabled = sbt.librarymanagement.Disabled
-  final val Binary = sbt.librarymanagement.Binary
-  final val Constant = sbt.librarymanagement.Constant
-  final val Full = sbt.librarymanagement.Full
-  final val Patch = sbt.librarymanagement.Patch
+  final val Disabled = sbt.librarymanagement.Disabled()
+  final val Binary = sbt.librarymanagement.Binary()
```

Comment:
Binary and Full were classes in sbt 0.13, but had lower case value `binary` and `full`.
So keeping `Binary` as the companion object of `sbt.librarymanagement.Binary` kind of makes sense to me.